Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:18.04
 
-# Set Virtuoso commit SHA to Virtuoso 7.2.5.2 release (16/08/2018)
-ENV VIRTUOSO_COMMIT 17c4ba1d5825822d8380ef8d9f978e2d57f4768f
+# Set Virtuoso commit SHA to Virtuoso 7.2.6.1 release (22/06/2021)
```

Comment:
```suggestion
# Set Virtuoso commit SHA to Virtuoso 7.2.6.1 release (2021-06-22)
```
Datetimes should *always*, especially when human facing, conform to ISO 8601 standard, i.e., YYYY-MM-DDTHH:mm:SS±00:00.